lib/parser.js

```diff
@@ -427,9 +438,26 @@ Parser.prototype._findBlockWithApiGetIndex = function(blocks) {
                 break;
             }
 
+            // check if the user want to filter by some specifique tag
```

Comment:
"wants" and "specific" (are you french? :p)

Reply:
hh yes :p

Reply:
Done